SEP-0002-revised.md

```diff
@@ -0,0 +1,262 @@
+
+It is expected that Project Members act as role models for the Community and are custodians of the SunPy Project.
+The Project Members elect a Steering Committee to lead the SunPy Project.
+
+#### Responsibilities and Rights
+
+Project Members may and should participate in formal votes on:
+
+* Adding new Project Members
+* Electing Steering Committee members
+* Appointing the Ombudsperson
+* Removing Project Members
+* Removing Steering Committee members
+* Removing the Ombudsperson
+* Approving changes to this SEP via a vote (see below)
+* Other matters where the Steering Committee or the Advisory Board believes a vote is appropriate.
+
+#### Membership
+
+Project Members of the SunPy Project must demonstrate all of the following:
+
+* A good grasp of the philosophy of the SunPy Project
+* A solid track record of being constructive and helpful
+* Significant contributions to the Project's goals, in any form
+* A willingness to dedicate time to improving the Project
+* A willingness to recruit, train, and nominate new team members
+* A track record of adhering to the project Code of Conduct.
+
+Project Membership acknowledges sustained and valuable efforts that align well with the philosophy and the goals of the SunPy Project.
+
+#### Initial Membership
+
+To be able to elect new Project Members an initial set of Project Members are needed.
+People meeting the following criteria will be invited to form the initial group of Project Members.
+They should consider the above responsibilities and actively agree to taking them on if they choose to accept the invitation.
+
+* Current commit access to the core repo, or any accepted affiliated package maintained by the project, and with a commit to that repository within the last 12 months.
+* Currently holds a role listed under the community roles section of the website: <https://sunpy.org/project/##community-roles>.
+* Is a currently serving Advisory Board member.
+
+#### Adding New Project Members
+
+Anyone is eligible to be nominated to be a Project member, as long as they meet the criteria laid out in the Membership section above.
+Self-nominations are encouraged.
+
+The process for adding a new Project member is as follows:
+
+1. A nomination is received by the Steering Committee, a nomination should include a short, factual, statement stating how the nominee meets the criteria for Project Membership.
+The Steering Committee, and the Ombudsperson check that the nomination is factually accurate and their record of contribution to the community adheres to the Code of Conduct.
+1. The candidate is then put to the Project Members.
+The name and nomination statement of the candidate will be shared with the Project Members.
+The candidate can become a Project Member if, during the two weeks following the candidate being put to the Project Members, five existing Project members approve the candidate and no Project Members call for an election.
```

Comment:
Why not just state that a new Project member can be added if they have 5 sponsors and then have a short probation period during which an election can be called? In practice, I think elections will almost never be used as it would be such a burden for someone to call for one.

My preference is that all Project Members need to be elected. That way everyone knows that they have the majority support of that community and it's fair and clear.

Reply:
The consensus at the coordination meeting was that we are happy with the existing text and we weren't clear on what change you are suggesting?